config/OWNERS

```diff
@@ -6,12 +6,14 @@ reviewers:
 - wojtek-t
 - chases2
 - dims
+- rjsadow
 approvers:
 - cblecker
 - spiffxp
 - wojtek-t
 - chases2
 - dims
+- rjsadow
```

Comment:
The process is to start in reviewers for a period, https://github.com/kubernetes/community/blob/master/community-membership.md

Reply:
(This allows us to get reviewers and to build ramping participation + trust)

I appreciate all the help but I prefer to stick to the documented approach and config/ covers A LOT 😅, brace for PRs :-)

Reply:
totally! I'll update.